crypto/bls.go

```diff
@@ -75,13 +75,22 @@ func (sk *PrKeyBLSBLS12381) Sign(data []byte, kmac hash.Hasher) (Signature, erro
 		return nil, errors.New("Sign requires a Hasher")
 	}
 	// check hasher output size
-	if kmac.Size() < opSwUInputLenBLSBLS12381 {
+	if kmac.Size() < minHashSizeBLSBLS12381 {
 		return nil, fmt.Errorf("Hasher with at least %d output byte size is required, current size is %d",
-			opSwUInputLenBLSBLS12381, kmac.Size())
+			minHashSizeBLSBLS12381, kmac.Size())
 	}
 	// hash the input to 128 bytes
 	h := kmac.ComputeHash(data)
-	return newBLSBLS12381().blsSign(&sk.scalar, h), nil
+
+	// intialize BLS context
+	_ = newBLSBLS12381()
```

Comment:
I think it makes sense to let `newBLSBS12381` return the internal error. Then to check the error here:
```
// returns a new BLS signer on curve BLS12-381
func newBLSBLS12381() (*blsBLS12381Algo, error) {
        ver err error
	once.Do(func() {
		blsInstance = &(blsBLS12381Algo{
			algo: BLSBLS12381,
		})
		err = blsInstance.init()
	})
         if err != nil {
            return nil, fmt.Error("could not initialize bls: %w", err)
         }
	return blsInstance, nil
}
```

Reply:
Thanks for pointing this out, I am planning to change the behaviour to `panic` instead of returning an error there since there might be no way to recover if there is an issue with `relic_init_BLS12_381`. 

 This will be done here: https://github.com/dapperlabs/flow-go/issues/4901